.github/workflows/test-tutorials.yml

```diff
@@ -17,3 +17,17 @@ jobs:
           git config --global user.email "test@example.com"
           git config --global user.name "Test"
           npx ts-node scripts/tutorial-runner.ts docs/zkapps/tutorials/01-hello-world.mdx
+
+  private-inputs-and-hash-functions:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v3
+        with:
+          # In case of version change, update README.md accordingly
+          node-version: 16
```

Comment:
We probably should update the node version here and with the other tutorials in CI because node 16 has reached end of life.

Reply:
Yep, @shimkiv has landed https://github.com/o1-labs/zkapp-cli/pull/479/

Reply:
what is the right update here?